src/game.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+function Game(){
+  // stores all the bowls
+  this.frames = [];
+};
+
+Game.prototype.scoreCard = function(){
+  return this.frames;
+};
+
+Game.prototype.addFrame = function(pins) {
+  //  pushes number of pins into array
+  this.frames.push(pins);
+};
+
+
+Game.prototype.score = function() {
+  var score = 0;
+  //  index of shots array starting at 0
+  var frameIndex = 0;
+ 
+  // iterate through the frames array, loop through 10 frames
+  for (var rollIndex = 0; rollIndex < 10; rollIndex++) {
+     
+
+    //  strike logic 
+      if (this.isAStrike(frameIndex)) {
+        score += this.addScoreOfTwoFrames(frameIndex) + this.frames[frameIndex + 2];  
```

Comment:
This is some nice clear logic again, I think implementing a scoring system at the end of the game would have been a better choice for me also to start making progress. good thinking